### ISSUE TITLE
reactiveTable error: argument is not an instance of Mongo.Collection, a cursor, or an array

### ISSUE THREAD

**Issue by sakshi-kataria** [aslagle/reactive-table#349]
When i am refreshing page

Template.customerFeedback.helpers({
  FeedBack: function () {
    if(Session.get('durationwise') === '1'){
      return FeedBack.find({$and:[{FeedBack_Value:{$ne: 'No-FeedBack'}},{"FeedBack_Date":{$gt:new Date(Date.now() - 24*60*60*1000)}}]},{sort:{FeedBack_Date: -1}});
    }else if(Session.get('durationwise') === '7'){
      return FeedBack.find({$and:[{FeedBack_Value:{$ne: 'No-FeedBack'}},{"FeedBack_Date":{$gt:new Date(Date.now() - 7*24*60*60*1000)}}]},{sort:{FeedBack_Date: -1}});
    }else if(Session.get('durationwise') === '30'){
      return FeedBack.find({$and:[{FeedBack_Value:{$ne: 'No-FeedBack'}},{"FeedBack_Date":{$gt:new Date(Date.now() -  30*24*60*60*1000)}}]},{sort:{FeedBack_Date: -1}});
    }
  },
  tableSettingsFeedBack() {
    return {
      rowsPerPage: 20,
      fields: [
        {key: 'FeedBack_Value', label: 'FeedBack', sortOrder: 1, sortDirection: 'ascending'},
        {key: 'FeedBack_Date', label: 'Date/Time', sortOrder: 0, sortDirection: 'descending'},
        {
          key: 'package_id', label: 'Parcel No.', sortOrder: 1, sortDirection: 'ascending',
          fn: function (value, object) {
          var _package= Packages.findOne(value);
          return `${_package.barcode} - (${_package.trackingNumber})`;
        }
      }]
    };
  }
});
Error: reactiveTable error: argument is not an instance of Mongo.Collection, a cursor, or an array is coming

**Comment by aslagle** [comment]
How is 'durationwise' set? When you refresh the page, Session.get('durationwise') might be undefined, so your FeedBack helper won't work.

**Comment by sakshi-kataria** [comment]
yes Session.get('durationwise')  is undefined but i am assigning value in this session in another helper

**Comment by aslagle** [comment]
I would guess the error message is because it's not set when this helper is first called. If the other helper sets it later, this helper should run again and show the table. If it's causing problems, you could have a helper that checks if durationwise is set and only calls reactiveTable if it is.
